layouts/partials/entry/meta/categories.html

```diff
@@ -2,10 +2,10 @@
 <span class="entry__meta-categories meta-categories">
 	<span class="meta-categories__list">{{ T "meta_categories" }}:
 	{{- range $index, $category := .Params.categories }}{{ if gt $index 0 }}, {{ end }}
-		<a class="meta-categories__link" href="{{ "categories/" | relLangURL }}{{ . | urlize | lower }}" rel="category">
+		<a class="meta-categories__link" href="{{ "categories/" | relLangURL }}{{ . | urlize | lower }}/" rel="category">
 			{{- . -}}
 		</a>
 	{{- end }}
 	</span>
 </span>
-{{- end }}
\ No newline at end of file
+{{- end }}
```

Comment:
Newline

Reply:
Git does this automatically, so it wasn't me.

Reply:
Could you fix this, please?

Tip: with `git add -p <file>` you can select changes for commit.

Reply:
I don't know, how to fix it. I just added the slash, and that's it. I didn't add/remove anything else and I don't touch this line at all.

Reply:
I understand you. Unfortunately, I don't know your environment, but if you in UNIX-like, then in most cases it's an editor thing. Use `truncate -s -1 <file>` command to trim eol, then commit changes and push.

Reply:
I think I know what's going on with this [last line](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_autocrlf). I'm using linux, so how to set this properly to avoid the issue?

Reply:
> I'm using linux, so how to set this properly to avoid the issue?

No, this option for Windows users in the most cases.

Just open terminal and run `truncate -s -1 <file>` to trim EOL from file. Then try to commit file again.

Reply:
Ok, did it work?